src/libertem/udf/base.py

```diff
@@ -684,6 +757,10 @@ def __init__(self, partition, idx):
     def get_locations(self):
         return self.partition.get_locations()
 
+    def get_resources(self):
```

Comment:
Short docstring would be nice here. The 1 means we need one CPU? What happens if I request 2 CPUs here? Will it raise an exception because it can't service that request?